includes/AnalysisSettings/StaticAnalysisSettings.js

```diff
@@ -0,0 +1,156 @@
+
+function StaticAnalysisType(type) {
+  const StaticAnalysisType_dict = {
+    undefined           : "GEOMETRICALLY_LINEAR",
+    "linear":             "GEOMETRICALLY_LINEAR",
+    "second order":       "SECOND_ORDER_P_DELTA",
+    "large deformations": "LARGE_DEFORMATIONS"
+  };
+
+  var SASType = StaticAnalysisType_dict[type];
+  if (SASType === undefined) {
+    SASType = "GEOMETRICALLY_LINEAR";
+    console.log("Wrong static analysis type input. Value was: " + type);
+    console.log("Correct values are: ('linear', 'second order', 'large deformations')");
+  };
+  console.log("Static analysis type: " + SASType);
+  return SASType;  
+}
+
+function NonlinearMethods(type, method) {
+
+  const nonlinearMethods_secondOrder_dict = {
+  "Picard"        : "PICARD",
+  "Postcritical"  : "NEWTON_RAPHSON_WITH_POSTCRITICAL_ANALYSIS",
+  "Newton"        : "NEWTON_RAPHSON"
+  };
+
+  const nonlinearMethods_largeDeformations_dict = {
+    "Picard"             : "PICARD",
+    "Combined"           : "NEWTON_RAPHSON_COMBINED_WITH_PICARD",
+    "Postcritical"       : "NEWTON_RAPHSON_WITH_POSTCRITICAL_ANALYSIS",
+    "Newton"             : "NEWTON_RAPHSON",
+    "Constant stiffness" : "NEWTON_RAPHSON_WITH_CONSTANT_STIFFNESS",
+    "Dynamic"            : "DYNAMIC_RELAXATION",
+  };
+
+  const nonlinearMehodsSwitcher = {
+    "Second-order (P-Δ)"   : nonlinearMethods_secondOrder_dict,
```

Comment:
this (P-Δ) could be potentially difficult to write on standard keyboard.

Reply:
David is using xx_yy if there is two-words argument see https://github.com/Dlubal-Software/Dlubal_JavaScript_Library/blob/71c34948bbec21d28ae6d98294e9279cfd4f21a1/includes/TypesForMembers/MemberEccentricity.js#L160

Reply:
> 

This parameter is inside FREM is has no interaction with user